mathics/packages/DiscreteMath/CombinatoricaV0.9.m

```diff
@@ -14,9 +15,10 @@
 350 Bridge Parkway, Redwood City CA 94065.  ISBN 0-201-50943-1.
 For ordering information, call 1-800-447-2226.
 
-These programs can be obtained on Macintosh and MS-DOS disks by sending
-$15.00 to Discrete Mathematics Disk, Wolfram Research Inc.,
-PO Box 6059, Champaign, IL 61826-9905. (217)-398-0700.
+These (and related) programs are available by anonymous ftp.cs.sunysb.edu
+in the pub/Combinatorica directory.  They can also be obtained on
+Macintosh and MS-DOS disks by sending $15.00 to Discrete Mathematics Disk,
```

Comment:
Really they still send you, in 2023, the code in a disk?

Reply:
It is what it says. It's not for me to change this.